packages/plugins/redux-plugin/package.json

```diff
@@ -16,6 +16,12 @@
     "redux": "^4.0.1"
   },
   "devDependencies": {
-    "@airy/maleo-gulp-plugin": "^1.0.0"
+    "@airy/maleo-gulp-plugin": "^1.0.0",
+    "@airy/maleo": "^1.0.0",
+    "gulp": "gulpjs/gulp.git#4.0",
+    "prop-types": "^15.6.2",
```

Comment:
actually `prop-types` shouldn't be included in the dependencies, since it's included with `react` deps